src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -76,76 +89,115 @@ public void setAddressBookFilePath(Path addressBookFilePath) {
         userPrefs.setAddressBookFilePath(addressBookFilePath);
     }
 
-    //=========== AddressBook ================================================================================
-
-    @Override
-    public void setAddressBook(ReadOnlyAddressBook addressBook) {
-        this.addressBook.resetData(addressBook);
-    }
+    //========== EntityListMethods ===============
 
-    @Override
-    public ReadOnlyAddressBook getAddressBook() {
-        return addressBook;
+    /**
+     * Returns the participant list located in the Model Manager.
+     *
+     * @return ParticipantList
+     */
+    public ParticipantList getParticipantList() {
```

Comment:
From the PoV of the Storage side, these lists should be sufficient for the Storage classes to work. Thanks for this.